iOS_Concepts.playground/Contents.swift

```diff
@@ -1,4 +1,54 @@
 import UIKit
+import Foundation
 
 
-var greeting = "Hello, playground"
+//Mark -  Model
+class StandardCalculatorModel {
+    func calculate(a: Int, b: Int, operation: String) -> Int {
```

Comment:
This is not a model , please read MVC design pattern to understand what a model is